### PR TITLE
Function prototype for different filters

### PR DESCRIPTION
**Pull request by Tushar7-coder**

### CODE REVIEW

File: helpers.h [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
+void edges(int height,int width,RGBTRIPLE **image);
 
 // Blur image
-// write protopype below this comment
+void blur(int height,int width,RGBTRIPLE **image);
 
 // Sepia image
-// write protopype below this comment
\ No newline at end of file
+void sepia(int height,int width,RGBTRIPLE **image);
```
[the remark this answers is not in the document]
Comment (chandrakant100) [reply helpers.h]: Are we going to use vectors?

**Review by chandrakant100 — CHANGES_REQUESTED** [review]

File: helpers.h [hunk 2]
```diff
@@ -1,16 +1,17 @@
 // To be Included the header file "bmp.h"
+#include "bmp.h"
 
 // Convert Image to grayscale
-// write protopype below this comment
+  void grayscale(int ,int ,RGBTRIPLE);
```
Comment (chandrakant100) [inline helpers.h:5]: Expected a double pointer in third argument i.3 `void grayscale(in, int, RGbTRIPLE **);`

Comment (chandrakant100) [inline helpers.h:5]: Expected a double-pointer in third argument i.3 `void grayscale(int, int, RGbTRIPLE **);` 
Also, add proper spacing as I did.
This should be followed in all function prototypes.

**Review by chandrakant100 — CHANGES_REQUESTED** [review]

**Comment by chandrakant100** [comment]
Everything looks great ! Let's wait for @Himanshu40 review.

File: helpers.h [hunk 3]
```diff
@@ -2,16 +2,16 @@
 #include "bmp.h"
 
 // Convert Image to grayscale
-void grayscale(int height,int width,RGBTRIPLE **image);
+  void grayscale(int ,int ,RGBTRIPLE);
 
 // Reflect image horizontally
-void reflect(int height,int width,RGBTRIPLE **image);
+  void reflect(int ,int ,RGBTRIPLE);
```
Comment (Himanshu40) [inline helpers.h]:
```suggestion
void reflect(int , int , RGBTRIPLE **);
```
Please, maintain what I have suggested.

**Review by Himanshu40 — APPROVED** [review]
Everything is fine.

File: helpers.h [hunk 4]
```diff
@@ -1,16 +1,17 @@
...
+ void edges(int, int, RGBTRIPLE **);
 
 // Blur image
-// write protopype below this comment
+ void blur(int, int, RGBTRIPLE **);
 
 // Sepia image
-// write protopype below this comment
\ No newline at end of file
+ void sepia(int, int,RGBTRIPLE **);
```
Comment (Himanshu40) [inline helpers.h:17]:
```suggestion
 void sepia(int, int, RGBTRIPLE **);
```
You didn't add space? Also please add a new line every time you reach the end of the file.

**Review by chandrakant100 — APPROVED** [review]

**Review by Himanshu40 — CHANGES_REQUESTED** [review]

**Review by chandrakant100 — APPROVED** [review]
Everything looks good! It is ready to merge.

**Review by Himanshu40 — APPROVED** [review]

**Pull request merged**